core/app/models/spree/shipping_method_stock_location.rb

```diff
@@ -0,0 +1,6 @@
+class ShippingMethodStockLocation
+  self.table_name = :spree_shipping_methods_stock_locations
```

Comment:
We could potentially start namespacing new models with Solidus. Not sure what @jhawthorn has in mind for the rename